Updated the process name that the installer checks for

isuftin opened the pull request:
To decide whether awslogs is already installed or not

benbridts committed:
Update metadata

benbridts committed:
Improve examples in readme

benbridts committed:
Use suggestions from foodcritic

benbridts committed:
Add chef ignore

benbridts committed:
Remove chef 12 specific attribute

benbridts committed:
Small whitespace fix in readme

benbridts committed:
Add gitignore

benbridts committed:
Increase version number

isuftin committed:
Updated the process name that the installer checks for to decide whether awslogs is already installed or not

wagnerone commented:
Thanks @isuftin 
I was puzzled for a bit by the constant reinstall of the awslog agent and came here to find this. I updated my local copy for the time being.

gibbsoft committed:
Add `create_if_missing` attribute to remote_file...

...resource to prevent trigger of unwanted upgrades on aws depending
upon the value of the boolean attribute ['cwlogs']['attempt_upgrade'].

benbridts committed:
Merge pull request #4 from alphagov/aws_download_installer_only_if_missing

Add `create_if_missing` attribute to remote_file...

atward committed:
Add proxy support

benbridts committed:
Merge pull request #5 from atward/proxy_support

Add proxy support

benbridts committed:
New release: add proxy support

brewboy reviewed (approved):
👍

isuftin committed:
Merge conflict fix

isuftin committed:
Ensure that awslogs agent does not attempt to be reinstalled if awslogs agent is already running

isuftin committed:
Removed needless system keyword

isuftin committed:
Added flexibility to how the CloudWatch agent is installed

isuftin committed:
Fix naming issue

isuftin committed:
Updated README.md with newest attributes
